packages/client/src/components/TestToolBox.js

```diff
@@ -60,23 +55,35 @@ const TestToolBox = ({ address }) => {
             <div className="mr-5">
               <label>
                 <strong>Balances</strong>
-                <a className="is-underlined ml-2" onClick={updateUserBalance}>
+                <button
+                  href="#"
+                  className="has-background-warning ml-2"
+                  onClick={updateUserBalance}
+                >
                   refresh
-                </a>
+                </button>
               </label>
               <ul>
                 <li>
                   <span>Account:</span>
-                  <ul className="ml-3">
-                    <li>Flow: {userFlowBalance}</li>
-                    <li>FUSD: {userFUSDBalance}</li>
-                  </ul>
+                  {userBalances && !!userBalances.length && (
+                    <ul className="ml-3">
+                      {userBalances.map((coin) => (
+                        <li key={COIN_TYPE_TO_META[coin.coinType].displayName}>
+                          {`${COIN_TYPE_TO_META[coin.coinType].displayName}: ${
+                            coin.balance
+                          }`}
```

Comment:
Would this work too?
```
`${COIN_TYPE_TO_META[coin.coinType].displayName}: ${coin.balance}`
```
Note: the only change is without the curly braces around the expression

Reply:
Tried to remove the curly braces, didn't work out, changing it to                           `{COIN_TYPE_TO_META[coin.coinType].displayName +": " + coin.balance}` for readability